.github/workflows/build_test.yml

```diff
@@ -0,0 +1,240 @@
+name: CI
+on:
+  pull_request_target:
```

Comment:
I also suggest running e2e tests when tags are pushed (`on: push: tags "*"` ), so the release could be validated.

Reply:
I'm assuming `on.push` just disregards `github.event.pull_request.labels` (hopefully)